src/com/palmergames/bukkit/towny/TownyFormatter.java

```diff
@@ -432,8 +436,9 @@ public static String getTime() {
 	}
 
 	/**
+	 * Gets the metadata of a Nation

```

Comment:
instead of metadata maybe say status screen

Reply:
Resolved in c27c7e04891a59b2778dd383f2b573495894c26d